plugins/ComposeAuth/src/commonMain/kotlin/io/github/jan/supabase/compose/auth/composable/NativeSignInResult.kt

```diff
@@ -0,0 +1,27 @@
+package io.github.jan.supabase.compose.auth.composable
+
+
+/**
+ * Response of Native SignIn
+ */
+sealed interface NativeSignInResult {
+    /**
+     * On Successful SignIn
+     * @param idToken
+     */
+    data class Success(val idToken: String? = null) : NativeSignInResult
```

Comment:
putting a idToken here could be confusing as you could think that you have to login yourself